operator/v1/types_ingress.go

```diff
@@ -1649,6 +1649,15 @@ type IngressControllerTuningOptions struct {
 	// +optional
 	TunnelTimeout *metav1.Duration `json:"tunnelTimeout,omitempty"`
 
+	// ConnectTimeout defines the maximum time to wait for
+	// a connection attempt to a server/backend to succeed.
+	//
+	// If unset, the default timeout is 5s.
```

Comment:
why isn't this the standard
>	// When omitted, this means the user has no opinion and the platform is left
>	// to choose reasonable defaults. These defaults are subject to change over time.
>	// The current default is 5s.

Reply:
Because the other timeouts in the tuning options use this phrasing, wanted to keep it consistent like if it was implemented this way from the very beginning (hence the bug).

Reply:
> Because the other timeouts in the tuning options use this phrasing, wanted to keep it consistent like if it was implemented this way from the very beginning (hence the bug).

Past mistakes rarely justify continuing those mistakes.  Let's update this to match the standard.

Reply:
Done.